src/SFA.DAS.Assessor.Functions/Functions/Ofqual/OrganisationsDownloader.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.Assessor.Functions.Domain.Entities.Ofqual;
+using SFA.DAS.Assessor.Functions.Domain.OfqualImport.Interfaces;
+
+namespace SFA.DAS.Assessor.Functions.Functions.Ofqual
+{
+    internal class OrganisationsDownloader : OfqualDownloader
+    {
+        public OrganisationsDownloader(IOfqualDownloadsBlobFileTransferClient blobFileTransferClient, IHttpClientFactory _httpClientFactory)
+            :base(blobFileTransferClient, _httpClientFactory.CreateClient("Organisations"), OfqualDataType.Organisations)
+        {
+        }
+
+        [FunctionName(nameof(DownloadOrganisationsData))]
+        public async Task<string> DownloadOrganisationsData([ActivityTrigger] string fileType, ILogger logger)
```

Comment:
Follow the same pattern of using 'unused' as the variable name if it's not used but required for the ActivityTrigger attribute

Reply:
I've been thinking of how to make this less clunky. Since I haven't decided on a solution I've named this `unused` while changing the type to `IDurableActivityContext` as part of the thought process.